aws/resource_aws_cloudformation_stack_set_test.go

```diff
@@ -447,6 +448,40 @@ func TestAccAWSCloudFormationStackSet_Parameters_NoEcho(t *testing.T) {
 	})
 }
 
+func TestAccAWSCloudFormationStackSet_PermissionModel_ServiceManaged(t *testing.T) {
+	var stackSet1 cloudformation.StackSet
+	rName := acctest.RandomWithPrefix("tf-acc-test")
+	resourceName := "aws_cloudformation_stack_set.test"
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t); testAccPreCheckAWSCloudFormationStackSet(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckAWSCloudFormationStackSetDestroy,
```

Comment:
Newer linter since this pull request was introduced:

```suggestion
		PreCheck:     func() { testAccPreCheck(t); testAccPreCheckAWSCloudFormationStackSet(t) },
		ErrorCheck:   testAccErrorCheck(t, cloudformation.EndpointsID),
		Providers:    testAccProviders,
		CheckDestroy: testAccCheckAWSCloudFormationStackSetDestroy,
```